jwql/website/apps/jwql/templates/about.html

```diff
@@ -9,46 +9,144 @@
+
+        API documentation can be found on <a href="https://jwql.readthedocs.io/en/latest/" target="_blank">readthedocs</a>.<br><br>
+
+
+
+
+        <h2 id="features">Features</h2><br>
+
+
+        <h4 id="visual_inspection">Visual Inspection</h4><br>
+
+        The JWQL application offers users a way to visually inspect JWST data that is present in the MAST archive.  Users may choose to view data for a particular instrument, program, and observation.  Users may also flag images as having one or more 'anomalies'.  Flagged anomalies are then stored in the JWQLDB database for future analysis.  Additionaly, users may view the contents of the FITS header for the given obeservation, as well as visually inspect other available filetypes.
 
         <div style='text-align: center'>
-            <img src={{ static('img/jwql_technologies.png') }}
-                 width=50%
-                 alt="JWQL technologies"
+            <img src={{ static('img/visual_inspection.png') }}
+                 width=100%
+                 alt="visual inspection feature example"
                  title="">
         </div><br>
 
-        The JWQL application is currently under heavy development. The 1.0 release is expected in the fall of 2021.<br><br>
 
-        API documentation can be found on <a href="https://jwql.readthedocs.io/en/latest/" target="_blank">readthedocs</a>.<br><br>
+        <h4 id="instrument_monitoring">Instrument Monitoring</h4><br>
 
-        Below is the list of automated monitors that the JWQL team has or is currently planning to develop, in collaboration with each instrument team.  Further information can be found on <a href="https://innerspace.stsci.edu/display/JWQLPROJ/Instrument+Monitor+Overview" target="_blank">Innerspace</a>.
+        The JWQL application automatically executes various instrument monitoring software on a daily basis and serves monitoring output plots and metadata.
+
+        <div style='text-align: center'>
+            <img src={{ static('img/instrument_monitor.png') }}
+                 width=100%
+                 alt="instrument monitor example"
+                 title="">
+        </div>
+
+        Below is the list of the monitors that the JWQL team has or is currently planning to develop, in collaboration with each instrument team.  Further information can be found on <a href="https://innerspace.stsci.edu/display/JWQLPROJ/Instrument+Monitor+Overview" target="_blank">Innerspace</a>.<br><br>
 
         <div style='text-align: center'>
             <img src={{ static('img/jwql_instrument_monitors.png') }}
                  width=100%
                  alt="JWQL instrument monitors"
                  title="">
+        </div>
+
+
+        <h4 id="engineering_database">Engineering Database</h4><br>
+
+        The JWQL application allows users to query the Engineering Database (EDB).  Users may search of mnemonics and plot a particular mnemonic over a given time period.  For more information about the EDB, see (some page)
```

Comment:
"search for" mnemonics

Some possible links to include in the "some page" placeholder:
https://outerspace.stsci.edu/display/MASTDOCS/Engineering+Data
https://outerspace.stsci.edu/display/MASTDOCS/Using+the+Engineering+Data+Portal

Reply:
also may be worth turning "Engineering Database" into a link to our EDB webpage